.gitignore

```diff
@@ -1,3 +1,5 @@
+.idea/
```

Comment:
I prefer to not have this in the local `.gitignore`, but your global `.gitignore`. 
https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/